README.md

```diff
@@ -0,0 +1,40 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+![A screenshot of the mobile version](./screenshot.png)
+
+> This is my future portfolio.
+
+This is one of the five portfolio templates offered by Microverse.
+
+## Built With
+
+- HTML/CSS
+- Git, Github, Lighthouse, Webhint, Stylelint.
+
+## Live Demo
+
+[https://jssol.github.io/portfolio/](https://jssol.github.io/portfolio/)
```

Comment:
- [OPTIONAL] 

- [ ] Good job so far 👍. Please consider fixing this link as it is broken. You can deploy your project to Github by following these [steps](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) or you could remove the live demo since it is a broken link.

Reply:
> Thanks, I'm about to put it live!